src/rules/no-named-default.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    return {
+      'ImportDeclaration': function (node) {
+        if (node.source == null) return
+
+        node.specifiers.forEach(function (im) {
+          if (im.type === 'ImportSpecifier' && im.imported.name === 'default') {
+            context.report({
+              node: im.local,
+              message: 'Use default import syntax to ' +
```

Comment:
Good thought! I'm used to an 80-char line length, this is definitely under the project's max of 99. :)